images/inventory-generator/root/usr/local/bin/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+OUTPUT_INVENTORY="generated_hosts"
+
+# generate inventory script
+/usr/local/bin/generate ${OUTPUT_INVENTORY}
+
+if [[ -z "${PLAYBOOK}" ]]; then
+  echo
+  cat $HOME/${OUTPUT_INVENTORY}
+  exit 0
+fi
+
+# run ansible playbook using generated inventory
+ansible-playbook -vvv -i ${OUTPUT_INVENTORY} ${PLAYBOOK}  --become --become-user root
```

Comment:
Double space before `--become`; missing new line.

Why do we default to `-vvv` (and hard code it)?

Why do we set `--become-user root`? It is the default.

Why do we need `--become`?